changelog/8124.misc.md

```diff
@@ -0,0 +1 @@
+Amend fixtures in tests that do not require model training.
```

Comment:
in my opinion we don't need a changelog as this isn't user facing

Reply:
It's a misc entry, so it shouldn't be included to the backlog anyway :)

Reply:
it actually does https://github.com/RasaHQ/rasa/blob/f52c12a069923ac7cd06fb65bb5b1d8f670fc2e1/CHANGELOG.mdx#L122-L121 . But you're right: doens't really matter 👍🏻

Reply:
Interesting that the `changelog/README.md` states that it won't be while in fact it's getting included :D need to fix that